packages/core/mx/contracts/escrow-factory/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+#![no_std]
+pub mod proxy;
+
+multiversx_sc::imports!();
+
+pub const STANDARD_DURATION: u64 = 8640000;
+
+#[multiversx_sc::contract]
+pub trait EscrowFactoryContract: proxy::StakingProxyModule {
+
+    #[init]
+    fn init(&self, staking: ManagedAddress) {
+        self.staking_contract_address().set(&staking);
+    }
+
+    #[endpoint(createEscrow)]
+    fn create_escrow(
+        &self,
+        token: TokenIdentifier,
+        token_bulk_max_value: BigUint,
```

Comment:
`BULK_MAX_VALUE` constant in solidity contracts exceeds the max value of a u64 constant that we use is mx rust sdk. As it will overflow and we cannot compile the contract, it is mandatory to set it in the storage as a big uint